src/scenes/AcceleratorRouter/Applications/ApplicationReviewModal.tsx

```diff
@@ -0,0 +1,183 @@
+    label: status,
+    value: status,
+  }));
+
+  const getApplicationReviewStatus = (status: ApplicationStatus): ApplicationReviewStatus => {
+    switch (status) {
+      case 'In Review':
+        return 'Submitted';
+      case 'Waitlist':
+        return 'Issue Pending';
+      default:
+        return status;
+    }
+  };
+
+  const [applicationReview, , onChange] = useForm<ApplicationReview>({
+    status: getApplicationReviewStatus(application.status),
+    feedback: application.feedback,
+    internalComment: application.internalComment,
+  });
+
+  const canUpdateStatus = useMemo(
+    () => statusOptions.find((status) => status === application.status) !== undefined,
+    [application.status]
+  );
+
+  const onCloseWrapper = useCallback(() => {
+    if (!(result?.status === 'pending')) {
+      onClose();
+    }
+  }, [onClose]);
+
+  const hasChange = useCallback(() => {
+    const originalReview: ApplicationReview = {
+      feedback: application.feedback,
+      internalComment: application.internalComment,
+      status: getApplicationReviewStatus(application.status),
+    };
+    return applicationReview !== originalReview;
+  }, [applicationReview, application]);
+
+  const onSave = useCallback(() => {
+    if (hasChange()) {
+      const dispatched = dispatch(
+        requestReviewApplication({ applicationId: application.id, review: applicationReview })
+      );
+      setRequestId(dispatched.requestId);
+    }
+  }, [dispatch, application, applicationReview, hasChange, setRequestId]);
+
+  useEffect(() => {
+    if (result && result.status === 'success') {
+      onSuccess();
+      onClose();
+      return;
+    }
+  }, [result, onSuccess, onClose]);
+
+  const busy = useMemo(() => result?.status === 'pending', [result]);
+
+  return (
+    <DialogBox
+      onClose={onCloseWrapper}
+      open={open}
+      title={strings.REVIEW_APPLICATION}
+      size='medium'
+      middleButtons={[
+        <Button
+          id='cancelReview'
+          label={strings.CANCEL}
+          disabled={busy}
+          priority='secondary'
+          type='passive'
+          onClick={onCloseWrapper}
+          key='cancel-button'
+        />,
+        <Button
+          id='saveNewApplication'
+          label={strings.SAVE}
+          disabled={busy || !hasChange()}
+          onClick={onSave}
+          key='save-button'
+        />,
+      ]}
+    >
+      <Grid container spacing={3} sx={{ padding: 0 }} textAlign='left'>
+        {canUpdateStatus && (
+          <Grid item xs={12}>
+            <Dropdown
+              fullWidth={true}
+              label={strings.APPLICATION_STATUS}
+              onChange={(value) => {
+                onChange('status', value);
+              }}
+              options={dropdownOptions}
+              required
+              selectedValue={applicationReview.status}
+            />
+          </Grid>
+        )}
+        {canUpdateStatus && (
```

Comment:
This could be part of the same previous condition block, all surrounded by a < >  ...  < / >